core/src/main/java/org/geysermc/geyser/GeyserImpl.java

```diff
@@ -302,6 +302,27 @@ private void startInstance() {
 
         Registries.RESOURCE_PACKS.load();
 
+        // Warnings to users who enable options that they might not need.
+        if (config.bedrock().useProxyProtocol()) {
+            logger.warning("Geyser is configured to expect HAProxy protocol for incoming Bedrock connections.");
+            logger.warning("If you do not know what this is, open the Geyser config, and set \"use-haproxy-protocol\" under the  \"bedrock\" section to \"false\".");
+        }
+
+        if (config.java().useProxyProtocol()) {
+            logger.warning("Geyser is configured to use proxy protocol when connecting to the Java server.");
+            logger.warning("If you do not know what this is, open the Geyser config, and set \"use-haproxy-protocol\" under the  \"java\" section to \"false\".");
+        }
+
+        if (config.advanced().disableXboxAuth()) {
+            logger.error("      / \\");
```

Comment:
Maybe the warning sign is too much? If we're going to do it, I'd want to make it one-time and then never again (could be solved by writing an empty file and checking if it exists on subsequent launches).

Reply:
https://github.com/GeyserMC/Geyser/pull/5390/commits/bf852cd449f4bb39fdae6f3d6a61af42cc712677
Removed the warning sign - however i'd still like to always show the warning due to the consequences that disabling this option brings with it

Reply:
That's fine with me.